Broken link for findlib

Trying to install ocamlfind from a clean build always fails, looks as though the camlcity.org domain is down:

`$ sudo add-apt-repository ppa:avsm/ppa-testing`
`$ sudo apt-get update -qq`
`$ sudo apt-get install -qq m4 ocaml ocaml-native-compilers camlp4-extra opam`
`$ export OPAMYES=1`
`$ opam init`
`$ opam install ocamlfind`

Fails with the following error message:

```
The following actions will be performed:
 - install ocamlfind.1.4.0
1 to install | 0 to reinstall | 0 to upgrade | 0 to downgrade | 0 to remove

=-=-= Installing ocamlfind.1.4.0 =-=-=
ocamlfind.1.4.0 Downloading http://download.camlcity.org/download/findlib-1.4.tar.gz
[ERROR] http://download.camlcity.org/download/findlib-1.4.tar.gz is not available


===== ERROR while installing ocamlfind.1.4.0 =====
Could not get the source for ocamlfind.1.4.0.
'opam install ocamlfind' failed.
```

--- a/packages/ocamlfind/ocamlfind.1.4.0/url
+++ b/packages/ocamlfind/ocamlfind.1.4.0/url
@@ -1,2 +1,2 @@
-archive: "http://download.camlcity.org/download/findlib-1.4.tar.gz"
+archive: "http://pkgs.fedoraproject.org/lookaside/extras/ocaml-findlib/findlib-1.4.tar.gz/5d1f8238c53964fdd14387b87b48b5d9/findlib-1.4.tar.gz"
 checksum: "5d1f8238c53964fdd14387b87b48b5d9"

